cmds/msr/msr.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// For now, this only works on linux.
+// +build linux
```

Comment:
I'll need to see if the name is preserved. Some x86 naming has crept into other architectures on linux. Checking ... nope, only on x86, good.